java/src/main/java/system/gameplay/engine/io/PlayerInputPrompter.java

```diff
@@ -0,0 +1,24 @@
+package system.gameplay.engine.io;
+
+import java.util.Scanner;
+
```

Comment:
This is very unprofessional on my part, but basically I wrote this class and then realized it might not be worth using because Lanterna probably already has stuff similar to what I wanted to add here. Since I had already put the time into writing and it and I wasn't sure if we'd need it I decided to keep it. In a real work environment you would scrap code that you didn't use whatsoever before merging your PR.